README.md

```diff
@@ -109,15 +140,25 @@ Your Answer:
 What are the differences between calling and referencing a function? Please provide examples of each.
 
 ```text
+Calling a function tells the browser to run the function. Referencing a function tells the browser which function to look for when it's called.
 
+Calling a function = runFunction();
+Referencing a function = runFunction
 ```
 ### Question #10
 
 Using the object literal notation, Define an object called student and give it the properties (your attributes) of name, age, and a method sayHello, that outputs "Hi, my name is [your_name]".
 
 Your Answer:
 ```js
-// write code here
+var student = {
+  name: 'Joe',
+  age: 28,
+  sayHello: function(){
+    console.log('Hi, my name is ' + student.name)
+  }
+}
+student.sayHello();
 ```
```

Comment:
:+1: This works, but ideally we would use `this.name` to reference the name in `sayHello`.